docs/docs/workflows/iam.md

```diff
@@ -0,0 +1,210 @@
+# IAM Configuration
```

Comment:
The workflow title and the content do not fit super well.

The page talks a lot about IAM, but it also explains how to generate resources and set values to the config that are not generated by `iam create`.

I think we could do two things:
1. Move all non-IAM related settings to a dedicated workflow
2. Rename this section to something more general "Cluster config and IAM"

I lean more towards 2. to keep it simple, although 1. has its merits if we think about a different persona (admin with extended permissions) creating IAM resources and another persona (DevOps engineer) creating the actual cluster.

Reply:
I also prefer option 2, as this generally refers to the same step in the setup (filling the `constellation-conf.yml`)

I think a DevOps engineer, setting up the cluster, will know if his access rights are insufficient for creating the IAM so he can forward this to an Administrator, on the other side and Administrator will know if the DevOps engineer can choose the machine types, etc.

Maybe this is my naive perspective - let me know what you think

Reply:
Makes sense, let's go with 2!